packages/xstate-wallet/src/tests-with-chain/chain.test.ts

```diff
@@ -83,11 +84,11 @@ it('correctly crafts a forceMove transaction (1x double-signed state)', async ()
   const outcome = simpleEthAllocation([
     {
       destination: playerA.destination,
-      amount: BigNumber.from(hexZeroPad('0x06f05b59d3b20000', 32))
+      amount: BN.from(hexZeroPad('0x06f05b59d3b20000', 32))
```

Comment:
```suggestion
      amount: BN.from('0x06f05b59d3b20000')
```
These don't need padding anymore